grpc.proto

```diff
@@ -119,3 +121,18 @@ message SignInit {
     repeated string party_uids = 3; // TODO replace this with a subset of indices?
     bytes message_to_sign = 4;
 }
+
+// Key presence check types
+message KeyPresenceRequest {
+    string key_uid = 1;
+}
+
+message KeyPresenceResponse {
+    enum Response {
+        present = 0;
+        absent = 1;
+        fail = 2;
```

Comment:
```suggestion
        RESPONSE_UNSPECIFIED = 0;
        RESPONSE_PRESENT = 1;
        RESPONSE_ABSENT = 2;
        RESPONSE_FAIL = 3;
```

Otherwise the linter over at the axelar-core repo won't allow this to go through